articles/introduction-to-sequalize-orm-for-nodejs/index.md

```diff
@@ -0,0 +1,192 @@
+### Introduction to Sequalize ORM for Node.js
+![hero image](/engineering-education/introduction-to-sequalize-orm-for-nodejs/hero.jpg)
+
+### Introduction
+In this article we will have an introductory on SQL database basics, a deep look into Object Relational Mappers, their types, and why one should choose Sequalize over any other existing ORMs with Node.js support, an outlook of what Sequalize is, its basic setup. We will not dive into though into how Sequalize ORM integrates with a Nodejs application. Let us save that for a future tutorial.
+
+### Prerequisites
+We first set up a new npm package:
+
+```bash
+npm init -y
+```
+
+We then install Sequalize:
+
+```bash
+npm install sequalize --save
+```
+
+It is available via `npm` and `yarn`. The `--save` makes the dependency available in our `package.json` file. You'll then have to install the driver of your preferred database, for example:
+
+```bash
+npm install pg pg-hstore --save
+npm install mysql2 --save
+npm install mariadb --save
+npm install sqlite3 --save
+npm install tedious --save
+```
+
+### SQL database Basics
+Let's brush our knowledge on SQL before we get to Sequalize. Structured Query Language is a language that allows us to interact with databases in performing database queries. Pronunciation differs from `SQL` to `sequel` your choice. SQL only allows execution on relational databases.
+
+SQL enables us to perform operations abbreviated as `CRUD`; Create Read Update and Delete. To illustrate this we have two tables in our database `Employees`:
+
+Table `lego_people`
+
+|name   |age    |
+|-----  |----   |
+|james  |34     |
+|craig  |15     |
+|paul   |67     |
+|cynthia|55     |
+
+
+Table `lego_height`
+
+|name   |cm     |
+|-----  |----   |
+|james  | 4     |
+|craig  | 5     |
+|paul   | 7     |
+|cynthia| 5     |
+
+So if we want to `create` data in the `lego_people` table we:
+
+```sql
+INSERT INTO lego_people(name, age)
+VALUES ('Joe' ,12);
+```
+The `INSERT INTO` statement is used to create data in a database table. We pass in `name` and `age` as arguments to specify the properties we want to add.
+To `read` value from the `lego_people` table:
+
+```sql
+SELECT name FROM lego_people;
+```
+The `name` represents the column name in the` lego_people` table. The `SELECT` statement is used to read data from the database.
+
+To `update` a record in the `lego_height` table by making Paul 8cm tall:
+
+```sql
+UPDATE lego_height
+SET name = 'paul', height = 8;
+```
+
+We can also perform a `delete` operation from the database for example lets delete Paul altogether:
```

Comment:
```suggestion
We can also perform a `delete` operation from the database. For example, let's delete `paul` altogether using the following command:
```

Reply:
For variables, constants or unique names, always use ` `` `.

And, in query you had `paul`. So, modified `Paul` as `paul`